bbq/opcode/instructions.yml

```diff
@@ -286,9 +286,28 @@
   controlEffects:
     - call:
 
-- name: "invokeDynamic"
+- name: "invokeMethodStatic"
```

Comment:
Good idea to include "method" in the instruction to indicate that this is used for object methods 👌